partialWaveFit/fitResult.cc

```diff
@@ -148,21 +328,31 @@ double
 fitResult::evidence() const
 {
 	double retval = 0.;
-	std::vector<double> summands = evidenceComponents();
+	const std::vector<double> summands = evidenceComponents();
 	for(unsigned int i = 0; i < summands.size(); ++i) {
 		retval += summands[i];
 	}
 	return retval;
 }
 
+
 std::vector<double>
 fitResult::evidenceComponents() const
 {
 	// make sure evidence can be calculated, i.e.:
```

Comment:
since the returned vector always seems to have 4 elements, wouldn't it make sense to initialize the four values with -infty in the beginning and explicitely set those that are defined?